File: bin/static/src/app/issuing-module/shared/model/configuration/master-coverage.ts
import { InputProperties } from '../configuration/input-properties';

/**
* this object represents a master coverage data for a specific product
*
* @author jtorres
*/
export class MasterCoverage {
    id: number;
    code: string;
    name: string;
    typeCoverage: string;
    category: string;
    active : boolean;
}

File: src/main/java/co/com/fsighttech/imp/issuing/bo/configuration/MasterCoverage.java
/**
 * 
 */
package co.com.fsighttech.imp.issuing.bo.configuration;

import lombok.Getter;
import lombok.Setter;

/**
 * This class represents the master of coverage in the application
 * 
 * @author jtorres
 */
@Getter
@Setter
public class MasterCoverage {

	private Long id;
	private String code;
	private String name;
	private String typeCoverage;
	private String category;
	private Boolean active;
	
	
}
